tark/tark_web/views.py

```diff
@@ -524,6 +525,15 @@ def mane_GRCh37_list(request):
     with connections['tark'].cursor() as cursor:
         cursor.execute(sql)
         results = ReleaseUtils.dictfetchall(cursor)
+	page = request.GET.get('page', 1)
```

Comment:
One minor edit:
I get this error when running the server
```
    page = request.GET.get('page', 1)
TabError: inconsistent use of tabs and spaces in indentation
```
Can you please fix the indentation and use spaces instead of tabs?

Reply:
apologies again @bilalebi , this is very embarrassing :(. I have fixed it now

Reply:
Thanks and no worries, I've done worse 🥲